docs/DeveloperGuide.adoc

```diff
@@ -264,28 +373,152 @@ Certain properties of the application can be controlled (e.g user prefs file loc
+This section contains some information or tools that may be useful for managing the project.
+
+=== Build Automation
+
+See <<UsingGradle#, UsingGradle.adoc>> to learn how to use Gradle for build automation.
+
+=== Continuous Integration
+
+We use https://circleci.com/[CircleCI, window="_blank"], https://travis-ci.org/[Travis CI, window="_blank"], and https://www.appveyor.com/[AppVeyor, window="_blank"_] to perform _Continuous Integration_ on our projects.
+See <<UsingCircleCI#, UsingCircleCI.adoc>>, <<UsingTravis#, UsingTravis.adoc>>, and <<UsingAppVeyor#, UsingAppVeyor.adoc>> for more details.
+
+=== Coverage Reporting
+
+We use https://coveralls.io/[Coveralls, window="_blank"] to track the code coverage of our projects. See <<UsingCoveralls#, UsingCoveralls.adoc>> for more details.
+
+=== Documentation Previews
+When a pull request has changes to asciidoc files, you can use https://www.netlify.com/[Netlify, window="_blank"] to see a preview of how the
+HTML version of those asciidoc files will look like when the pull request is merged. See <<UsingNetlify#, UsingNetlify.adoc>> for more details.
+
+=== Making a Release
+
+Here are the steps to create a new release.
+
+.  Update the version number in link:{repoURL}/src/main/java/seedu/address/MainApp.java[`MainApp.java`, window="_blank"].
+.  Generate a JAR file <<UsingGradle#creating-the-jar-file, using Gradle>>.
+.  Tag the repo with the version number, e.g. `v0.1`.
+.  https://help.github.com/articles/creating-releases/[Create a new release using GitHub, window="_blank"] and upload the JAR file you created.
+
+=== Managing Dependencies (Check this!!!)
+
+A project often depends on third-party libraries. For example, PowerLifting Competition Coordinator depends on the http://wiki.fasterxml.com/JacksonHome[Jackson library, window="_blank"_] for XML parsing.
+We make use of Gradle to manage the project dependencies. Gradle can automate the downloading of these dependencies, saving us time and effort.
+
+While strongly discouraged by us, you can nevertheless opt for one of the following alternatives.
+
+* Include those libraries in the repo (though this bloats the repo size).
+* Download those libraries manually (though this creates extra work for you).
 
 [appendix]
 == Product Scope
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
+* organizes small to medium sized powerlifting competitions
+* has a need to manage a significant number athletes during the competition
+* has a need to see the ranking of the participants of each competition
+* prefer desktop applicationss over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: manage powerlifting competitions faster than a typical mouse/GUI driven app
 
 [appendix]
 == User Stories
```

Comment:
The user story format is correct and the 3 parts are all present. The benefit matches the function nicely. Overall everything is in order and there is no missing user stories. Good job!